internal/repository/repository.go

```diff
@@ -34,12 +34,12 @@ type Repository struct {
 }
 
 // New returns a new repository with backend be.
-func New(be restic.Backend) *Repository {
+func New(be restic.Backend, minpacksize uint) *Repository {
 	repo := &Repository{
 		be:     be,
 		idx:    NewMasterIndex(),
-		dataPM: newPackerManager(be, nil),
-		treePM: newPackerManager(be, nil),
+		dataPM: newPackerManager(be, nil, minpacksize),
+		treePM: newPackerManager(be, nil, minpacksize),
```

Comment:
Here, you could simply add a `minPacksize : minpacksize` and use later.
I would also propose to make a new method `func (r *Repository) MinPackSize() uint` which returns this default.
This can then be easily used in the `prune` newimplementation.

Reply:
>`minPacksize : minpacksize`

That's much more sensible.

> new method

Probably a good idea, allows us to have more interoperability.